### ISSUE TITLE
Duplicate job requests

### ISSUE THREAD

**Issue by iaindillingham** [opensafely-core/job-server#2594]
Two very similar job requests were created, three seconds apart. [The earlier job request][2] failed, reporting `__error__`. [The later job request][3] succeeded. The stack trace isn't available through [Sentry][1] but it is available through Honeycomb's [Job Trace page][5]. It mentions `assert_new_jobs_created`, which @madwort may be investigating. See also opensafely-core/job-runner#517.

@lucyb and I discussed the issue in [this Slack thread][4].

[1]: https://ebm-datalab.sentry.io/issues/2629669572/?project=5443358&referrer=slack
[2]: https://jobs.opensafely.org/lshtm/long-term-kidney-outcomes-after-sars-cov-2-infection/long_term_kidney_outcomes_after_covid/14953/uda5yjgfxge33pmv/
[3]: https://jobs.opensafely.org/lshtm/long-term-kidney-outcomes-after-sars-cov-2-infection/long_term_kidney_outcomes_after_covid/14954/zvhf5jt7uyvkzruj/
[4]: https://bennettoxford.slack.com/archives/C0270Q313H7/p1675557576841149
[5]: https://ui.honeycomb.io/bennett-institute-for-applied-data-science/environments/production/datasets/jobrunner/trace/mCjukpiBvQy?fields[]=c_name&fields[]=c_service.name&span=76e301c1ad366d5a

**Comment by lucyb** [comment]
This issue occurred again [here](https://bennettoxford.slack.com/archives/C0270Q313H7/p1675698457875629).

Looking at the `Created by the backend` field in Job Server on the Job Request page, in both instances the duplicate jobs were created within one second.

Testing locally, I've found that it's possible to submit a job request twice by double-clicking the Submit button on the `job_request_create.html` page. 

I've tried swapping the base template to the new tw version and using the new button component, but that seems to have the same issue too.

As a first step, we should stop users from double-clicking that Submit button.
